apps/sentry-client-desktop/src/features/home/SentryWallet.tsx

```diff
@@ -138,56 +139,55 @@ export function SentryWallet() {
 		}
 	}
 
-	function getKeys() {
-		if (nodeLicenseStatusMap.size === 0) {
+	function getAddresses() {
+		if (sentryAddressStatusMap.size === 0) {
 			return (
 				<tr className="flex pr-8 py-[15px] bg-nulnOil text-sm">
-					<td colSpan={3} className="w-full text-center text-lg text-medium text-americanSilver">No keys found.</td>
+					<td colSpan={3} className="w-full text-center text-lg text-medium text-americanSilver">No addresses found.</td>
 				</tr>
 			);
 		}
 
-		let i = 0;
 		const element: Array<ReactNode> = [];
 
-		new Map([...nodeLicenseStatusMap].filter(([, status]) => {
-			if (selectedWallet === null) {
-				return true;
-			}
-			return status.ownerPublicKey === selectedWallet;
-		}))
-			.forEach((status, key) => {
-				//const isEven = i++ % 2 === 0;
-
+		sentryAddressStatusMap.forEach((status: SentryAddressInformation, key: string) => {
+			if (selectedWallet === null || status.address === selectedWallet) {
 				element.push(
-					<tr className={`bg-nulnOil flex pl-[25px] pr-8 text-sm border-b border-chromaphobicBlack`} key={`license-${i}`}>
-						<td className="w-full max-w-[65px] pr-4 py-4 text-lg font-medium text-elementalGrey">{key.toString()}</td>
-						<td className="w-full max-w-[400px] pr-4 py-4 text-lg font-medium text-elementalGrey">{status.ownerPublicKey}</td>
+					<tr className={`bg-nulnOil flex pl-[25px] pr-8 text-sm border-b border-chromaphobicBlack`} key={`address-${key.toString()}`}>
+						<td className="w-full max-w-[215px] pr-4 py-4 text-lg font-medium text-elementalGrey whitespace-nowrap">
+							{status.isPool ? <><img className="w-[20px] h-[20px] inline rounded-full mr-1" src={status.logoUri}></img> {status.name}</> : `${key.slice(0, 6)}...${key.slice(-6)}`}
+						</td>
+						<td className="w-full max-w-[108px] pr-4 py-4 text-lg font-medium text-elementalGrey"><KeyIcon extraClasses="inline fill-[#A19F9F]" /> {status.keyCount}</td>
 						<td className="w-full max-w-[400px] px-4 py-4 text-lg font-medium text-elementalGrey">
 							{status.status}
 						</td>
 					</tr>
 				);
-			})
+			}
+		});
+
 		return element;
 	}
 
 	function getWalletCounter() {
+
+		if (loading) {
+			return <>Loading...</>
+		} else if (sentryAddressStatusMap.size == 0) {
+			return <>No wallets...</>
+		}
+
+		let walletCounter = `${sentryAddressStatusMap.size} of ${owners.length} wallets/pools`;
```

Comment:
I think this line may need to have the text adjusted? It says owner's length, but the text after says "wallets/pools"

What do you think about something like this => `let walletCounter = ${sentryAddressStatusMap.size} wallets/pools across ${owners.length} owners;`

Reply:
This was only for testing, the UI will change and this text won't be displayed like that, we will have an Icon and just show the numbers with a tooltip I believe. Should not have been part of this story, slipped through when testing locally.

I don't think this has any consequences but please let me know if this should be reverted.